src/geh_common/databricks/databricks_api_client.py

```diff
@@ -24,6 +24,23 @@ def get_job_id(self, job_name: str) -> int:
                     return job.job_id
         raise Exception(f"Job '{job_name}' not found.")
 
+    def get_job_run_id(self, job_id: int, active_only: bool = True) -> int:
```

Comment:
Why are you satisfied with just any of potentially multiple running instances?

Reply:
In our context, we know that we only have 1 running instance, as this is a rule for the job itself.

We would just like to get the running job. Though I actually want to remove the `raise Exception`, because it is OK if the job has no running jobs.

Reply:
Maybe the method should be renamed to `first_job_run_id`?

Reply:
Ok, but that's not our context when we use this client. I think you should choose between returning all ids, and reflecting the fact in the name of the function. E.g. `get_any_job_run_id()`. Could also be "first" instead of "any" as in Linq. But that may lead to think that it's the job instance that were started first.

Reply:
the list received is in descending order based on start time

Reply:
Works for me 👍

Reply:
Done. Ready for re-review :-)